packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -365,7 +365,6 @@ export class BlockNoteEditor<
           ...newOptions._tiptapOptions?.editorProps?.attributes,
           ...newOptions.domAttributes?.editor,
           class: mergeCSSClasses(
-            "bn-root", // TODO: remove this class?
```

Comment:
@matthewlipski was this safe to remove like this? because it is used in editor.css